core/org.eclipse.cdt.core/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.cdt.core; singleton:=true
-Bundle-Version: 8.0.0.qualifier
+Bundle-Version: 8.0.100.qualifier
```

Comment:
The version has already been bumped since the last release, so this version bump isn't needed.